server/devdocs/bin/generate-components-usage-stats.js

```diff
@@ -8,9 +8,10 @@
 
 var async = require( 'async' ),
 	camelCase = require( 'lodash/camelCase' ),
-	config = require( 'config' ),
+	config = require( '../../config' ),
```

Comment:
No, it doesn't break. I probably made this change before I figured out how `NODE_PATH` works. I've just reverted to remove some noise.